### ISSUE TITLE
[BUG] All known Phoenix-1 Helicopter Bugs

### ISSUE THREAD

**Issue by StealthyRiot** [xXJamieXx/paradox-bugs#133]
**Your Discord Name**
Stealthy_Riot#9382

**Describe the bug**
Below are all the current bugs I'm aware of in Phoenix-1, I don't think anything here is a high priority. 
1. Radar Lock HUD can get stuck on screen. Must switch back onto camera to fix. This can probably be deferred since it's easily fixed in city.
2. MRPD spawning local helicopters on Helipad. [Still occurring as of 3/26/2022]
3. [Enhancement] Camera Lock reads in Kilometers, would like it to read Miles Per Hour
4. [Enhancement] Could use a place to pull out a Heli in the Sandy area. Only two locations I've found are MRPD & Paleto PD. 
5. [Enhancement] Takes too long to be able to lock onto a vehicle, and camera sensitivity is too low
6. [Enhancement] Have radar lock stop if no visual on suspect for x amount of time. For now, pilots are taught to break off their own lock if they lose sight.

[FIXED] Altitude appears to read off Sea Level not Ground Level. Not sure if this is normal.
[FIXED] Co-Pilot's camera altitude is double the pilot's altitude reading. I believe the co-pilot's camera altitude is correct
[FIXED] Stress/ Seatbelt Issues
[FIXED] Sniper spawning for passengers in back. (Mostly fixed, just shows some ammo now)

**To Reproduce**
Radar staying locked
1. Just lock onto a vehicle and fly away or watch it go under tunnels. Lock will not be lost.

Radar Lock HUD frozen
1. Unknown [Video Provided]

Attempting to Radar lock a suspect
1. Go into camera by pressing "E"
2. Zoom in a little on vehicle and hover mouse over vehicle
3. High Speed Racers are practically impossible to lock onto with how long it takes and the sensitivity

 MRPD Spawning local Helicopters
1. Go to the MRPD roof, I typically see a local PD Heli or Buzzard. Makes landing back at MRPD difficult. Must land at the back Helipad and impound the local helicopter. then switch back and store Heli. 

**Screenshots**
Radar staying locked
https://www.twitch.tv/videos/1263010965

Radar Lock HUD Frozen
https://www.twitch.tv/videos/1263010968

local Heli spawning
https://www.twitch.tv/videos/1265695947

**Comment by StealthyRiot** [comment]
I have updated the list as of 3/26/2022. Appreciate all the work you guys have put into this! I changed a couple of these from bugs to enhancements. At this point Phoenix 1 is in a good place, but I would really like to be able to get accurate speed readings on a suspect's vehicle. Locking onto racers mid race is still practically impossible unless they crash or stay traveling straight on a highway.
